### DISCUSSION TITLE
Bind Aqara H1 Single Rocker to Sonoff zbmini

### DISCUSSION THREAD

**Discussion by AcidSleeper** [Koenkk/zigbee2mqtt#17448] [Devices]
Hey

Im lost. Binding in Z2M is almost impossible and I like some guide how to bind a remote (Aqara H1) to a switch (Sonoff ZBmini).

I have tried to bind in every way possible (with and without group, direct in remote and more so but when I try to press bind I only get “Failed to bind”. So Im out of ideas…

Please HELP! =)

**Comment by AcidSleeper** [comment]
Anyone?

**Comment by AcidSleeper** [comment]
BUMP

**Reply by lerra** [reply]
Did you manage to solve it?

**Reply by AcidSleeper** [reply]
No I didnt.

**Reply by 5arer** [reply]
I think it just doesn't work together. Same with ZHA.
Aqara rocker E1 (identified as lumi.remote.acn003 in ZHA) doesn't want to bind to zbmini2. But IKEA tradfri battery switch binds successfully so it sounds it's problem with Aqara.
